tests/transports-integrations/tests/conftest.py

```
@@ -0,0 +1,159 @@
+"""
+Pytest configuration for integration-specific tests.
+"""
+
+import pytest
+import os
+from typing import Dict, Any
```

Ruff (0.11.9)

`typing.Dict` is deprecated, use `dict` instead

(UP035)